db/view/view_update_generator.hh

```diff
@@ -75,6 +76,8 @@ public:
 
     replica::database& get_db() noexcept { return _db; }
 
+    const sharded<service::storage_proxy>& get_storage_proxy() const noexcept {return _proxy;};
```

Comment:
Why const? The storage proxy is not part of the state of the view update generator, so it is fine to return mutable reference.

Reply:
I didn't modify anything so I added the const, but I didn't think about it much, will remove.